README.md

```diff
@@ -28,12 +28,47 @@ automatically navigate the app and reach your desired outcome.
+
+- **Unclear word choice**: Consider whether or not the goal you created could be interpreted in multiple ways. If so, consider rewriting or adding clarifying statements.
+- **Incorrect path taken**: ScriptIQ uses common sense reasoning to generate the test using the intent of your goal. But sometimes ScriptIQ takes the wrong path, making an assumption about the best way to accomplish your goal which is incorrect. When an incorrect step is taken, consider defining the correct action explicitly in your goal. For example, if ScriptIQ attempts to login when you do not require it to login, you can add "do not login" to your goal.
+- **Context provided in assertions**: The assertion statements do not guide ScriptIQ towards accomplishing the goal, they only check whether or not the descriptions provided in the assertion match seen screens. If you've only included necessary context in the assertion fields you'll need to add it to the goal.
+- **Non-journey based goal**: ScriptIQ is based on GUI interactions, it cannot reference the source code, so goals which refer to the source code might not be understood.
+
+Here is an example of updating your goal:
+
+1. Create your first, general goal: ![ScriptIQ example goal which did not work](media/imgs/example_scriptiq_incorrect.png)
+2. You can see that it did not accomplish the goal of logging in because it scrolled down when the option to login is brought up when you click "More".
+3. Update the goal with this additional context and run again: ![ScriptIQ example goal which did work](media/imgs/example_scriptiq_correct.png)
+4. The updated goal is now accurately finding the login button and clicking it.
+
+Those are just some examples of why a goal might not work and how to fix it. Feel free to be creative!
+
+## How Do Assertions work?
+
+Once you've started generating tests, you might want to more quickly determine if your test succeeded or failed, this is where the _Assert Inputs_ fields come in. Assert Inputs are descriptions of screens which are used to evaluate the test generated. When assertions are provided, the LLM will determine if the assertion matches what is on the screen for each assertion provided against every screen visited. These assertions can describe the entire screen (e.g. login screen) or describe an element on the screen (e.g. page with checkout button). These assertions are _not_ used to generate test steps, any intent needed to drive the test generation must be included in the _Test Goal_ field. Below you can see examples of assertions and an example output.
```

Comment:
This is a single line with 758 characters 😅 Would it be possible to break it down into 80 character chunks? Last time I checked with the team, that seemed like that preferred line length, especially when you try to place windows side by side. Easier to view them inside editors that don't have word-wrap enabled. Also easier to review any changes made in the future!

PS: This is true for some of the other lines above as well 🙏

Reply:
Yes! I can do that!